deploy/helm/magda/charts/registry-api/templates/_helpers.tpl

```diff
@@ -51,14 +51,21 @@ spec:
 {{- if .root.Values.global.enableLivenessProbes }}
         livenessProbe:
           httpGet:
-            path: "/v0/ping"
+            path: /v0/status/live
             port: 80
-          initialDelaySeconds: 60
+          initialDelaySeconds: 10
```

Comment:
Why reduce this to 10? The reason this is set to 60 is that it sometimes takes > 30 seconds to become responsive, which causes it to be restarted, which then causes it to take > 30 seconds to start, which causes it to be restarted etc. At the very least it needs to be configurable I would think?

Admittedly the reasons behind these could use some commenting.